### ISSUE TITLE
Is 2.x still the recommended version to start with?

### ISSUE THREAD

**Issue by tinymachine** [lerna/website#72]
https://github.com/lerna/website/blob/28fa6c926c2093804c1db3cc01362c89dc9f9efa/index.html#L63

Not sure if this needs an update or you're really suggesting installing 2.x instead of 3.x. If you could clarify, I'd appreciate it. Thanks!

**Comment by evocateur** [comment]
No, 2.x is not recommended anymore. v3 has been the preferred version for quite awhile. I never update the docs here because I always forget it exists, definitely need a better solution that is more tightly integrated with the actual code...

**Comment by tinymachine** [comment]
Thanks for clarifying!

**Comment by tad-lispy** [comment]
I guess it's 4 now :P

**Comment by JamesHenry** [comment]
Hi Folks 👋 

You may or may not know that lerna is now under the stewardship of Nrwl (announcement here https://github.com/lerna/lerna/issues/3121), a company with a long history of not just producing valuable open-source software (OSS), but also backing others (at the time of writing, Nrwl has donated over $50,000 to OSS it _hasn't_ created, see https://opencollective.com/nx for full details).

Quite simply, Nrwl ❤️  OSS, and is committed to making lerna the best it can be. We use it ourselves.

We hope you will continue to be a part of this community as we look to take things forward from here!

Please see https://github.com/lerna/lerna/discussions/3140 for more details on our plans for 2022.

We have just pushed out a new version of the website, which we will continue to iterate on, we hope you like it and find it useful!

Feel free to open a new issue on here if you spot anything that needs fixing or improving.

Many thanks 🙏
